doc/nrf/releases_and_maturity/releases/release-notes-3.1.0.rst

```diff
@@ -0,0 +1,1183 @@
+
+IDE, OS, and tool support
+=========================
+
+* Updated the required `SEGGER J-Link`_ version to v8.42.
+* Removed the separate requirement for installation of the `nRF Util's device command <Device command overview_>`_ from the :ref:`install_ncs` page under :ref:`installing_vsc`.
+  The tool and the command are now included in the |NCS| toolchain bundle.
+
+  .. note::
+
+     If you plan to work with command line, you still need to download nRF Util and install the ``sdk-manager`` command in order to get the toolchain bundle.
+
+Board support
+=============
+
+* Added:
+
+  * Bias-pull-up for Thingy:91 X nRF9151 UART RX pins.
+  * Alternative partition tables for Thingy:91 X.
+
+Bootloaders and DFU
+===================
+
+* Added:
+
+  * MCUboot support as first-stage immutable bootloader using hardware cryptographic acceleration on nRF54L10 and nRF54L05.
+  * MCUboot support for hardware downgrade protection the nRF54L Series devices.
+  * MCUboot support for SHA512 signature verification the nRF54L Series devices.
+  * Linking Time Optimization (LTO) support for MCUboot the nRF54L Series devices.
+  * Experimental MCUboot support for AES-256 image encryption for the nRF54L Series devices.
+  * Experimental MCUboot support as first-stage immutable bootloader for the nRF54H20:
+
+    * Direct XIP mode with merged images for application and radio cores.
+    * Swap mode with separated images for application and radio cores.
+
+  * MCUboot sample optimized for NVM footprint and implementing recommended settings when working with the nRF54L15 DK.
+    The :ref:`mcuboot_minimal_configuration` sample utilizes hardware cryptography for ED25519 signature verification and KMU for secure key storage, achieving an NVM footprint of ~16.5 KB.
+  * Added support to provision KMU on nRF54L115, nRF54L10, and nRF54L05 devices.
+
+* Updated the following security features:
+
+  * HW downgrade prevention.
+  * RAM memory cleanup controlled by the :kconfig:option:`CONFIG_MCUBOOT_CLEANUP_RAM` Kconfig option.
+  * ECIES-X25519 encryption improvements.
+
+* Fixed several stability and functionality improvements.
+
+* Removed:
+
+  * SUIT support.
+  * suit-generator.
+  * suit-processor.
+
+Developing with nRF54L Series
+=============================
+
+* Added:
+
+  * The :ref:`ug_nrf54l_dfu_config` documentation page, describing how to configure Device Firmware Update (DFU) and secure boot settings using MCUboot and NSIB.
+  * The :ref:`ug_nrf54l_ecies_x25519` page on enabling the AES encryption with ECIES-X25519, which is used for secure encryption key exchange.
+
+* Updated by increasing the default value of the :kconfig:option:`CONFIG_MPSL_HFCLK_LATENCY` Kconfig option to support slower crystals.
+  See the Kconfig description for a detailed description on how to select the correct value for a given application.
+
+Developing with nRF54H Series
+=============================
+
+* Removed SUIT support.
```

Comment:
Move the documentation updates here. They are so big and significant that they should be highlighted here.